iphone/Classes/TiUIButton.m

```diff
@@ -263,7 +263,7 @@ -(void)setBackgroundImage_:(id)value
 {
 	[backgroundImageCache release];
 	RELEASE_TO_NIL(backgroundImageUnstretchedCache);
-	backgroundImageCache = [[TiUtils loadBackgroundImage:value forProxy:[self proxy]] retain];
+    backgroundImageCache = [[self loadImage:value] retain];
     self.backgroundImage = value;
```

Comment:
Please fix indentation of both lines 266 and 267 to conform with the one of 265.